docs/src/prototypes/popups/helper.ts

```diff
@@ -0,0 +1,77 @@
+import * as _ from 'lodash'
```

Comment:
Would rename this to dataMocks, or anything else more related to `data`. The name helpers associate more to some utils function, at least in my opinion. What do you think?